source/sass/type.scss

```diff
@@ -103,6 +103,10 @@ h4,
   font-weight: 400;
 
   @include scaleText(24px, 28px, 28px, 36px);
+
+  a {
```

Comment:
to fix
... <img width="691" alt="screen shot 2018-05-01 at 12 14 28 pm" src="https://user-images.githubusercontent.com/2896608/39489201-9500bada-4d39-11e8-9e3a-a7904246ca5d.png">